modules/recon/hosts-hosts/ipinfodb.py

```diff
@@ -7,16 +7,18 @@ class Module(BaseModule):
     meta = {
         'name': 'IPInfoDB GeoIP',
         'author': 'Tim Tomes (@lanmaster53)',
-        'version': '1.1',
+        'version': '1.2',
         'description': 'Leverages the ipinfodb.com API to geolocate a host by IP address. Updates the \'hosts\' table '
                        'with the results.',
         'required_keys': ['ipinfodb_api'],
         'query': 'SELECT DISTINCT ip_address FROM hosts WHERE ip_address IS NOT NULL',
         'options': (
             ('limit', True, True, 'toggle rate limiting'),
```

Comment:
Might it make more sense to remove the `limit` option and just allow people to specify 0 seconds if they don't want to rate limit?

Reply:
I do believe it would and made the change.